tests/compute/test_broadcast.py

```diff
@@ -0,0 +1,57 @@
+import dgl
```

Comment:
How about put all the testcases in `test_broadcast`, `test_softmax` and `test_topk` in `test_readout`?

Reply:
I think it's not appropriate to call `broadcast` and `softmax` readout operations.

Reply:
Fixed.